### ISSUE TITLE
pip install add tests folder to site-packages

### ISSUE THREAD

**Issue by Matresh** [omarryhan/aiogoogle#39]
After install the tests folder are added to site-packages which may cause conflicts  in imports

**Comment by omarryhan** [comment]
Hey Matresh, are you installing directly from the git repo or from Pypi (via pip)?

Because in the manifest.in file, I excluded the tests folder and on my machine, the tests directory isn't included.

I think it would be better if we exclude in one place i.e. manifest.in. As per [this]([shown here](https://stackoverflow.com/questions/608855/excluding-a-top-level-directory-from-a-setuptools-package#comment39374906_9769754), wouldn't adding `recursive-exclude tests*` to the manifest.in be a better solution?

**Comment by Matresh** [comment]
Yeah, Using Pypi:
pip install aiogoogle

Using manifest.in will be better i think.

**Comment by Matresh** [comment]
I never worked with manifest.in but if I'm correctly understand prune must be similar to recursive-exclude tests* but prune currently not working

**Comment by omarryhan** [comment]
Hmm, weird. Can you try and see if `recursive-exclude tests*` works for you instead of `prune tests`. If it does, I can merge that.

**Comment by Matresh** [comment]
Im not sure how can i test it, directly pip install from git repo is representative?
Or need to upload into PyPi ?

**Comment by Matresh** [comment]
I have 2 packages aiogoogle and test in top_level.txt

And 2 packages in -site-packages

![image](https://user-images.githubusercontent.com/36481677/103419056-08d29f80-4b9a-11eb-96e0-18035105f523.png)
![image](https://user-images.githubusercontent.com/36481677/103419078-24d64100-4b9a-11eb-8e3b-44ea7ce70fb9.png)

**Comment by omarryhan** [comment]
This is weird.

> Im not sure how can i test it, directly pip install from git repo is representative?

Yes. Clone it, then make your changes. Then, in the CLI, write: `python setup.py sdist`. Then , you'll find the new package in the `dist` directory.

I did just that right now, and added the line that you wanted to add to the `setup.py` file and it didn't change anything. I also tried replacing `prune` with `global-exclude` and replacing `tests/` with `**/*tests` and also nothing.

All of the methods I tried didn't include the tests directory in the `.tar` package. Also, all of them included the line `tests` in the `top_level.txt` file.

Are you installing directly from Github to get this error?

If adding `exclude=["tests"]` is the only thing that works for you, I don't mind merging it.
